rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/Sw360ResourceServer.java

```diff
@@ -168,6 +176,36 @@ public OpenAPI customOpenAPI() {
                 .info(new Info().title("SW360 API").license(new License().name("EPL-2.0")
                                 .url("https://github.com/eclipse-sw360/sw360/blob/main/LICENSE"))
                         .version(restVersionString))
-                .servers(List.of(server));
+                .servers(List.of(server))
+                .path("/health", new PathItem().get(
+                        new Operation().tags(Collections.singletonList("Health"))
+                                .summary("Health endpoint").operationId("health")
+                                .responses(new ApiResponses().addApiResponse("200",
+                                        new ApiResponse().description("OK")
+                                                .content(new Content()
+                                                        .addMediaType("application/json", new MediaType()
+                                                                .example("""
+                                                                        {
+                                                                          "status": "UP",
```

Comment:
May be, it would look good to extract it to a constant

Reply:
Done :+1: